lib/common/socket/evloop.c.h

```diff
@@ -447,7 +449,7 @@ h2o_socket_t *h2o_evloop_socket_accept(h2o_socket_t *_listener)
     if ((fd = cloexec_accept(listener->fd, (struct sockaddr *)peeraddr, peeraddrlen)) == -1)
         return NULL;
     fcntl(fd, F_SETFL, O_NONBLOCK);
-    sock = &create_socket_set_nodelay(listener->loop, fd, H2O_SOCKET_FLAG_IS_ACCEPTED_CONNECTION)->super;
+    sock = &create_socket_set_nodelay(listener->loop, fd, H2O_SOCKET_FLAG_IS_ACCEPTED_CONNECTION, peeraddr->sa_family)->super;
```

Comment:
I think we can:
* call `create_socket` directly here, and
* then call a utility function that does `if (sa_family == AF_INET || sa_family == AF_INET6) setsockopt(TCP_NODELAY)`.